test/extended/baseline/baseline.go

```diff
@@ -0,0 +1,18 @@
+package baseline
+
+import (
+	"time"
+
+	g "github.com/onsi/ginkgo"
+)
+
+// This test is intended to be run by itself against a new cluster to collect
+// baseline performance data.
+var _ = g.Describe("[sig-scalability][Suite:openshift/scalability] Managed cluster should", func() {
+	defer g.GinkgoRecover()
+
+	g.It("approach fixed resource consumption in the steady state", func() {
+		g.By("observing the cluster at rest for 15 minutes after installation")
```

Comment:
Install-complete isn't really important here, is it?  If you are going to look for stable resource consumption, it can just be "in the 15m after we launch this test", regardless of whether you launch immediately post-install or 10d after creating a cluster, or whatever.

Reply:
I think what you're saying makes sense. The 15m window could be at any point in this cluster's post-installation lifecycle. But it does seem to have to start after installation is complete, or at least guarantee to capture a significant amount of the post-installation after test execution. So post-install seems like a pretty easy constraint. The execution time beyond the install completion seems irrelevant. Agree?

Reply:
I am fine assuming that no test runs before install-complete.  And if someone decides to run this test-case during installation, it would be appropriate if it failed due to resource growth.  I don't think we need to make that part of the test-case name.  But if we wanted to make it `at any point after installation`, I'd be ok with that too.